### DISCUSSION TITLE
WebUI didn't support training(fine tune) for model SD21?

### DISCUSSION THREAD

**Discussion by terrificdm** [d8ahazard/sd_dreambooth_extension#487] [Q&A]
When I changed my model from SD20(768-v-ema.ckpt) to SD21(v2-1_768-ema-pruned.ckpt), it seemed that "train" function of webui is not working properly anymore(no matter textual inversion or hypernetwork)... the "Loss:nan" occurred. Did anyone try fine tune with model SD21 before, success? Any suggestion for this? Thanks.

**Comment by terrificdm** [comment]
Sorry, seemed posed to the wrong place...

**Comment by FugueSegue** [comment]
Has anyone been able to train a face in the Dreambooth extension using any SD v2.1 chpt? I've been trying all day without success.
